salt/client/ssh/__init__.py

```diff
@@ -554,6 +560,7 @@ def run(self):
                     outputter,
                     self.opts)
 
+        sys.exit(final_exit)
 
 class Single(object):
```

Comment:
There should be two blank lines above this class declaration. That should make the pylint test failure go away.